notebooks/nlp_notebook_tutorial.ipynb

```diff
@@ -346,9 +346,9 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.7.3"
+   "version": "3.9.7"
   }
  },
  "nbformat": 4,
- "nbformat_minor": 1
+ "nbformat_minor": 4
```

Comment:
is this file supposed to be changed in this PR? probably not worth your time to restore but just pointing it out in case there were supposed to be more changes or something

Reply:
This happened because i ran the notebook, it seems it didn't get stripped out by our script or me clearing the outputs. It's harmless.